file_manager/tests/tests_units.py

```diff
@@ -45,7 +45,7 @@ def test_delete_file(self):
 
     def test_no_such_file(self):
         with self.assertRaises(FileDoesNotExistException):
-            FileManager.delete_file(file_id=1, user=self.user)
+            FileManager.delete_file(file_id=-1, user=self.user)
```

Comment:
why -1?

Reply:
Because the file with such id definitely does not exist

Reply:
ok